### DISCUSSION TITLE
Is it possible to use c# language to implement the function

### DISCUSSION THREAD

**Discussion by HotObsidian** [naverz/zepeto-studio-global#73] [Q&A]
Zepeto uses TypeScript language, but I want to implement some functions in C# language, will it cause audit problems?

**Comment by naverz-mkao** [comment]
Hi HotObsidian,

You can ONLY use C# for in-editor scripts. C# will not actually compile when you make a build.

**Reply by HotObsidian** [reply]
Sorry,I don't understand what you mean，Do you mean that the function I implemented with c# script will fail after the game is uploaded?

**Reply by naverz-mkao** [reply]
Hi HotObsidian.

Yes, the  C# script will fail when you build your unity project. To be exact, C# doesn't actually fail, it is just that C# is ignored by the compiler.

C# scripts only work while the game runs in the editor. For example, if you hit the "play" button to run the game in the editor, C# will work. Once you build though, C# is ignored by the compiler.
